go.mod

```diff
@@ -1,15 +1,14 @@
 module github.com/juju/terraform-provider-juju
 
-go 1.23
+go 1.23.6
 
-toolchain go1.23.4
+toolchain go1.24.0
```

Comment:
Maybe drop this to 1.23.6 to match the go version?

Reply:
done